cmd/wallet-js-sdk/src/didcomm/didcomm.js

```diff
@@ -187,6 +190,8 @@ export class DIDComm {
     },
     { from, timeout } = {}
   ) {
+
+    console.debug("in didcomm.initiateCredentialShare() - about to call proposePresentation for '"+this.user+"', myLabel: '"+myLabel+"', routerConnections: '"+routerConnections+"', reuseConnection:'"+reuseConnection+"', reuseAnyConnection:'"+reuseAnyConnection+"', connectionTimeout:'"+connectionTimeout+"'")
```

Comment:
logs

Reply:
yeah i kept these to help troubleshooting

Reply:
removed all console.debug/log calls added in this change